### PR TITLE
Story4

### PR DESCRIPTION
**Pull request by JoshThomasField**

### CODE REVIEW

File: script.js [hunk 1]
```diff
@@ -1,103 +1,130 @@
-// Get the modal
+const ghost = document.querySelector('.ghost');
+const pumpkin = document.querySelector('.pumpkin');
+const reaper = document.querySelector('.grim_reaper');
+const lineEight = document.querySelector('.line_eight');
```
Comment (norayna) [inline script.js]: Using the name `lineEight` in this context, it's not immediately obvious what this is. Upon discussion with the team I've found out this is the eighth segment of the path, however in this case it happens to be the part of the path where the Grim Reaper's movement is triggered. 

Therefore we've concluded that the specific HTML element should have a new class added to it called `grim_reaper_zone` or something like that, and that that class should be used in the JavaScript instead, so it's clear what this element is.

File: script.js [hunk 2]
```diff
@@ -1,103 +1,130 @@
...
+    {transform: 'translateY(0px)'}
+], {
+    duration: 3000,
+    iterations: Infinity
+});
+reaperMove.pause();
 
-document.querySelector('#closeWinModal').addEventListener('click', () => {
+// code to make splash screen mouse hover
+function homeScreenGhost () {
```
Comment (norayna) [inline script.js]: Both the comment and the function name don't really explain what this does - change to something like `addSplashScreenGhostAnimation()`

**Review by norayna — CHANGES_REQUESTED** [review]

File: script.js [hunk 3]
```diff
@@ -1,103 +1,130 @@
...
+    winningDoor.removeEventListener('mouseenter', win);
+    winModal.style.display = "block";
+    ghostMove.pause();
+    pumpkinMove.pause();
+    reaperMove.cancel();
+    lineEight.removeEventListener('mouseleave',moveReaper);
+}
+
+// code here to activate lose modal displaying
+function lose() {
```
Comment (mporam) [inline script.js:76]: this function appears to be identical to the `win` function apart from which modal is opened. to make it more DRY we could have a `reset` function that does all the common functionality, and then call that function from the `win` and `lose` functions with the line to show the correct modal.

**Review by mporam — CHANGES_REQUESTED** [review]

File: script.js (hunk 2 above)
Comment (norayna) [reply script.js]: Also in this case it's not really worth creating a function for this just to call it once immediately after - you should just run the code without having it in a function. Reasons why this is okay:
- The code runs only once
- This project is very front-endsy and we're not doing unit testing on it (unusual project)

File: script.js [hunk 4]
```diff
@@ -42,48 +42,41 @@ const reaperMove = reaper.animate([
 reaperMove.pause();
 
 // code to make splash screen mouse hover
```
Comment (norayna) [inline script.js]: Change this comment to be more clear about what exactly happens

**Review by norayna — CHANGES_REQUESTED** [review]
Inconsistent naming conventions for CSS classes and ids - some are `likeThis`, some `like-this`, some `like_this`. You can choose which convention to follow but it's best if they are all the same.

**Review by norayna — APPROVED** [review]

**Review by mporam — APPROVED** [review]

**Pull request merged**